### PR TITLE
Fixed #8571 (Required Sign Asterisk)

### LINKED ISSUE #8571: The asterisk sign is missing in the admin email configuration fields.
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

master

### Issue Description

The asterisk sign is missing in the admin email configuration fields.

### Preconditions

1. Admin should be logged in.

### Steps to reproduce

1. Go to 'Admin panel'
2. Click on 'Configuration'
3. Scroll down to 'Email setting'
4. See the error

### Expected Result

It should be shown the asterisk sign for the required fields.

### Actual Result

The asterisk sign is missing. [Here](https://prnt.sc/LGei-tx0PxW9)

### Screenshots

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by JunaidJaved248**
## Issue Reference
#8571 

## Description
Just Changed the Field-type blade file, used existing rendered validation in file for required sign setup

## How To Test This?
Go to 'Admin panel'
Click on 'Configuration'
Scroll down to 'Email setting'
It showing the asterisk sign for the required fields.

## Documentation
- [ ] My pull request requires an update on the documentation repository.

### CODE REVIEW

**Comment by devansh-webkul** [comment]
Hi @JunaidJaved248,

We have other plans to solve this and we are thinking of a long-term perspective so we need a permanent fix. Not a quick fix.

Hint:
Just go through the `system.php` file and connect all validations with the frontend and backend as well.

**Comment by JunaidJaved248** [comment]
> Hi @JunaidJaved248,
> 
> We have other plans to solve this and we are thinking of a long-term perspective so we need a permanent fix. Not a quick fix.
> 
> Hint: Just go through the `system.php` file and connect all validations with the frontend and backend as well.

@devansh-webkul , I've employed the system.php validation and examined other files where similar validation checks are applied. Specifically, in the system.php file related to email configuration, I noticed the validation rule 'required|max:50' being used.

I've replicated this approach in the frontend to determine if a field's validation includes the 'required' rule. If it does, I display an asterisk next to that field.

Alternatively, I'm considering introducing a new key called 'is_required' with values 'true' or 'false'. This would allow us to easily check this requirement in the frontend.

What would be your recommendation regarding this approach?

**Comment by devansh-webkul** [comment]
For the front end, we are using vee validate 4, my approach would be to connect with the vee validate 4 so that if any validation comes it will automatically attached to vee validate 4. Rest all validation will get handled by vee validate 4. If there is some customized validation then I can think of that.

For backend it is already Laravel stuff so easy one.

**Comment by devansh-webkul** [comment]
Right now, you are using is_required. If tomorrow there are some more validation. We need to handle it as per that perspective.

**Comment by JunaidJaved248** [comment]
@devansh-webkul I have reviewed again with others like create customer, 
This Page Validation process is same as other form like customer,
As per reference, you can see that for showing asterisk with label of field, Just added class="required",
I just used same scenario for it, For fixing this issue,

I just shown asterisk on based on $validation from `system.php` file. It seems to fine to me. If still something there that i am skipping now.

**Comment by JunaidJaved248** [comment]
@devansh-webkul Can you update on it?

File: packages/Webkul/Admin/src/Resources/views/configuration/field-type.blade.php [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
     $validations = $coreConfigRepository->getValidations($field);
 
+    $is_required = Str::contains($validations, 'required') ? 'required' : '';
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/configuration/field-type.blade.php:12]: Make it camelcase.

**Comment by JunaidJaved248** [comment]
@devansh-webkul Its Done

**Review by devansh-webkul — APPROVED** [review]

**Review by Amitk-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**